com.github.you_win.VPupPr.yml

```diff
@@ -0,0 +1,50 @@
+id: com.github.you_win.VPupPr
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: OpenSeeFaceGD.x86_64
+finish-args:
+    - "--socket=x11"
+    - "--share=ipc"
+    - "--share=network" # to set up the deps for openseeface
+    - "--device=all"
+    - "--socket=pulseaudio"
+    - "--filesystem=home:ro"
```

Comment:
that is a lot of filesystem permissions.

Reply:
I've limited it to what I can, so as to not break the user experience and have to tinker with the flatpak permissions too much. I've set the home folder to read-only, so it's not as dangerous, I hope.

Reply:
why do you need to read the home directory?